inventory/byo/hosts.origin.example

```diff
@@ -530,8 +530,8 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # list of options please see roles/openshift_logging/README.md
 #
 # Configure loggingPublicURL in the master config for aggregate logging, defaults
-# to https://kibana.{{ openshift_master_default_subdomain }}
-#openshift_hosted_logging_hostname=https://kibana.example.com
+# to kibana.{{ openshift.common.dns_domain }}
```

Comment:
https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/openshift_hosted.yml#L23 is the default logic used elsewhere